### ISSUE TITLE
Python Permission Denied - Develop dd376be

### ISSUE THREAD

**Issue by BKSteve** [SickChill/SickChill#7603]
Latest update to develop Edit: dd376be4e5a29d105ae98b2c82e0a1ca3a2939fd

Gives permission denied and invalid version errors.
Running SC on SynoDSM with `sickchill_x64-6.1_20210729-22.spk`

**Comment by BKSteve** [comment]
Well that was weird.
I'd logged into the DSM as `root` then did `sudo -u sc-sickchill /bin/bash` which produced the above.

Re ssh with different used the 'sudo -u sc-sickchill /bin/bash` produced the following different log.
```
sudo -u sc-sickchill /bin/bash
Password:
bash-4.3$ /volume1/@appstore/sickchill/env/bin/python3 /volume1/@appstore/sickchill/share/SickChill/SickChill.py --config /volume1/@appstore/sickchill/var/data/config.ini --datadir /volume1/@appstore/sickchill/var/data
Checking poetry
Can write to /var/packages/sickchill/target/.local/lib/python3.8/site-packages: False
Can write to /volume1/@appstore/sickchill/env/lib/python3.8/site-packages: True
pip installed: True
Found wheelhouse dir at /volume1/@appstore/sickchill/share/wheelhouse
pip args: /volume1/@appstore/sickchill/env/bin/python3 -m pip install --no-input --disable-pip-version-check --no-python-version-warning --no-color -qU -f/volume1/@appstore/sickchill/share/wheelhouse setuptools poetry wheel
poetry installed: True
Warning: The lock file is not up to date with the latest changes in pyproject.toml. You may be getting outdated dependencies. Run update to update them.

  InvalidVersion

  Invalid PEP 440 version: '3.6.'

  at /volume1/@appstore/sickchill/env/lib/python3.8/site-packages/poetry/core/version/pep440/parser.py:67 in parse
       63│     @classmethod
       64│     def parse(cls, value: str, version_class: Optional[Type["PEP440Version"]] = None):
       65│         match = cls._regex.search(value) if value else None
       66│         if not match:
    →  67│             raise InvalidVersion(f"Invalid PEP 440 version: '{value}'")
       68│
       69│         if version_class is None:
       70│             from poetry.core.version.pep440.version import PEP440Version
       71│

The following error occurred when trying to handle this error:


  ValueError

  Could not parse version constraint: >=3.6.*

  at /volume1/@appstore/sickchill/env/lib/python3.8/site-packages/poetry/core/semver/helpers.py:139 in parse_single_constraint
      135│
      136│         try:
      137│             version = Version.parse(version)
      138│         except ValueError:
    → 139│             raise ValueError(
      140│                 "Could not parse version constraint: {}".format(constraint)
      141│             )
      142│
      143│         if op == "<":
Restarting SickChill with ['/volume1/@appstore/sickchill/share/SickChill/.venv/bin/python3', '/volume1/@appstore/sickchill/share/SickChill/SickChill.py', '--config', '/volume1/@appstore/sickchill/var/data/config.ini', '--datadir', '/volume1/@appstore/sickchill/var/data']
Checking poetry
Can write to /var/packages/sickchill/target/.local/lib/python3.8/site-packages: False
Can write to /volume1/@appstore/sickchill/share/SickChill/.venv/lib/python3.8/site-packages: True
pip installed: True
Found wheelhouse dir at /volume1/@appstore/sickchill/share/wheelhouse
pip args: /volume1/@appstore/sickchill/share/SickChill/.venv/bin/python3 -m pip install --no-input --disable-pip-version-check --no-python-version-warning --no-color -qU -f/volume1/@appstore/sickchill/share/wheelhouse setuptools poetry wheel
poetry installed: True
Found wheelhouse dir at /volume1/@appstore/sickchill/share/wheelhouse
pip args: /volume1/@appstore/sickchill/share/SickChill/.venv/bin/python3 -m pip install --no-input --disable-pip-version-check --no-python-version-warning --no-color -qU -f/volume1/@appstore/sickchill/share/wheelhouse appdirs==1.4.4 babelfish==0.6.0; python_version >= "3.6" and python_version < "4.0" backports.zoneinfo==0.2.1; python_version < "3.9" and python_version >= "3.6" and (python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6") beautifulsoup4==4.10.0; python_full_version > "3.0.0" beekeeper-alt==2021.7.16; python_full_version >= "3.6.2" and python_version < "4" bencode.py==4.0.0 cachecontrol==0.12.7; python_version >= "3.6" certifi==2021.10.8; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6" cffi==1.15.0; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6" chardet==4.0.0; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.5.0") charset-normalizer==2.0.7; python_full_version >= "3.6.0" and python_version >= "3.6" click==8.0.3; python_version >= "3.6" cloudscraper==1.2.58 colorama==0.4.4; python_version >= "3.6" and python_full_version < "3.0.0" and platform_system == "Windows" or platform_system == "Windows" and python_version >= "3.6" and python_full_version >= "3.5.0" configobj==5.0.6 cryptography==35.0.0; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6" decorator==5.1.0; python_version >= "3.6" deluge-client==1.9.0 deprecated==1.2.13; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6" dogpile.cache==1.1.4; python_version >= "3.6" enzyme==0.4.1 feedparser==6.0.8; python_version >= "3.6" future==0.18.2; python_version >= "2.6" and python_full_version < "3.0.0" or python_full_version >= "3.3.0" gntp==1.0.3 greenlet==1.1.2; python_version >= "3" and python_full_version < "3.0.0" and (platform_machine == "aarch64" or platform_machine == "ppc64le" or platform_machine == "x86_64" or platform_machine == "amd64" or platform_machine == "AMD64" or platform_machine == "win32" or platform_machine == "WIN32") and (python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.6.0") or python_version >= "3" and (platform_machine == "aarch64" or platform_machine == "ppc64le" or platform_machine == "x86_64" or platform_machine == "amd64" or platform_machine == "AMD64" or platform_machine == "win32" or platform_machine == "WIN32") and (python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.6.0") and python_full_version >= "3.5.0" guessit==3.3.1 html5lib==1.1; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.5.0") httplib2==0.20.1; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.4.0") idna==3.3; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6" ifaddr==0.1.7 imagesize==1.2.0; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.4.0") imdbpy==2021.4.18 importlib-metadata==4.8.1; python_version < "3.8" and python_version >= "3.6" and (python_version >= "3.6" and python_full_version < "3.0.0" and python_version < "3.8" or python_full_version >= "3.6.0" and python_version < "3.8" and python_version >= "3.6") importlib-resources==5.3.0; python_version < "3.7" and python_version >= "3.6" ipaddress==1.0.23 js2py==0.71 jsonrpclib-pelix==0.4.3.1 kodipydent-alt==2021.7.16; python_full_version >= "3.6.2" and python_version < "4" lxml==4.6.3; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" mako==1.1.5; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.4.0") markdown2==2.4.1; python_version >= "3.5" and python_version < "4" markupsafe==2.0.1; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6" msgpack==1.0.2; python_version >= "3.6" new-rtorrent-python==1.0.1a0 oauthlib==3.1.1; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6" packaging==20.9; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.4.0") pbr==5.6.0; python_version >= "3.6" profilehooks==1.12.0 putio.py==8.7.0 pyaes==1.6.1 pycparser==2.20; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6" pygithub==1.55; python_version >= "3.6" pyjsparser==2.7.1 pyjwt==2.3.0; python_version >= "3.6" pymediainfo==5.1.0; python_version >= "3.6" pynacl==1.4.0; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" and python_version >= "3.6" pynma==1.0 pyopenssl==21.0.0; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.6.0") pyparsing==2.4.7; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" pysocks==1.7.1; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.4.0") pysrt==1.1.2 python-dateutil==2.8.2; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.3.0") python-slugify==5.0.2; python_version >= "3.6" python-twitter==3.5 python3-fanart==2.0.0 pytz-deprecation-shim==0.1.0.post0; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6" pytz==2021.3 qbittorrent-api==2021.8.23 rarfile==4.0 rebulk==3.0.1 requests-oauthlib==1.3.0; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.4.0" requests-toolbelt==0.9.1 requests==2.26.0; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.6.0") send2trash==1.8.0 sgmllib3k==1.0.0; python_version >= "3.6" six==1.16.0; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6" soupsieve==2.2.1; python_version >= "3.6" and python_full_version > "3.0.0" sqlalchemy==1.4.26; (python_version >= "2.7" and python_full_version < "3.0.0") or (python_full_version >= "3.6.0") stevedore==3.5.0; python_version >= "3.6" subliminal==2.1.0 text-unidecode==1.3; python_version >= "3.6" timeago==1.0.15 tmdbsimple==2.8.0 tornado==6.1; python_version >= "3.5" tus.py==1.3.4 tvdbsimple==1.0.6 typing-extensions==3.10.0.2; python_version < "3.8" and python_version >= "3.6" tzdata==2021.5; platform_system == "Windows" and python_version >= "3.6" and (python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version >= "3.6") tzlocal==4.0.2; python_version >= "3.6" unidecode==1.3.2; python_version >= "3.5" urllib3==1.26.7; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.6.0" and python_version < "4" and python_version >= "3.6" validators==0.18.2; python_version >= "3.4" webencodings==0.5.1; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" win-inet-pton==1.1.0 wrapt==1.13.2; python_version >= "3.6" and python_full_version < "3.0.0" or python_full_version >= "3.5.0" and python_version >= "3.6" xmltodict==0.12.0; python_full_version >= "3.6.2" and python_version < "4" zipp==3.6.0; python_version < "3.7" and python_version >= "3.6"
2021-10-31 12:51:44 INFO :: MAIN :: 392226: Unable to find IMDb show info in the database
2021-10-31 12:51:47 INFO :: MAIN :: 408325: Unable to find IMDb show info in the database
2021-10-31 12:51:47 INFO :: MAIN :: Starting SickChill [develop] using '/volume1/@appstore/sickchill/var/data/config.ini'
2021-10-31 12:51:47 INFO :: WEBSERVER :: Starting SickChill on http://0.0.0.0:8081/
2021-10-31 12:51:47 INFO :: CHECKVERSION :: Checking for updates using SOURCE
```
And running

**Comment by BKSteve** [comment]
Leave open for develop branchers until fix is in.

**Comment by BKSteve** [comment]
Didn't double check before closing, Dependabot overwrote the fix again in #7610, see #7597 for original if anybody interested.
